crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -1,10 +1,16 @@
 # Nushell Environment Config File
 
 def create_left_prompt [] {
+    let home = ($env | get -i HOME | into string)
```

Comment:
This is not cross platform. Windows does not have `$env.HOME`. Windows uses `$env.USERPROFILE`